ArrayEx/House.java

```diff
@@ -0,0 +1,26 @@
+package ArrayEx;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by LewisMonplaisir on 17/07/2017.
+ */
+public class House {
+    public void HouseTypes() {
+
+        List<Long> priceRange = new ArrayList<Long>();
+
```

Comment:
The priceRange variable should have been given a plural name like priceRanges becuase it is a collections of many price range.